### ISSUE TITLE
Change to using rates instead of absolute numbers

### ISSUE THREAD

**Issue by ydhuang28** [reichlab/covid19-forecast-hub#1922]
Right now the map and the time chart use absolute numbers. Since different jurisdictions have wildly different population numbers, perhaps the better way to visualize numbers would be to use a rate, i.e., case/mortalities per 1K/10K/100K population.

This is open to suggestions!

**Comment by RobertWalraven** [comment]
That would definitely make it possible to see another dimension of the data on the visualization site.   Perhaps a checkbox could be added to the configuration panel to allow scaling by population.

**Comment by jarad** [comment]
Agreed. The map hasn't been that useful (other than selecting a location) for a long time. Rather than cumulative rate maybe it would be better to visualize percentage increase, so that we could see what locations are getting worse the fastest.

**Comment by nickreich** [comment]
Are we talking about updating only the map or the timechart too? I think adding this for the map (or a toggle to view rate/count) would be valuable, but I don't think it's worth the hassle to support this for the timechart.

**Comment by jarad** [comment]
Just the map as the map is where we are comparing locations and thus the rate is relevant. (At least that is what I'm thinking.)

**Comment by nickreich** [comment]
To be consistent with our weekly reports, let's use outcome per 100k population. so
for incident deaths it would be `last_obs_inc_death/population*100000`
for cumulative deaths it would be `last_obs_cum_death/population*100000`
for inc cases it would be `last_obs_inc_cases/population*100000`
